pkgs/top-level/all-packages.nix

```diff
@@ -13671,6 +13671,8 @@ with pkgs;
 
   systembus-notify = callPackage ../applications/misc/systembus-notify { };
 
+  systemctl-tui = callPackage ../applications/misc/systemctl-tui { };
```

Comment:
Please use `pkgs/by-name` and remove this line

Reply:
Cool! I didn't know about this new structure.